stdlib/seq.mli

```diff
@@ -298,7 +298,8 @@ val for_all2 : ('a -> 'b -> bool) -> 'a t -> 'b t -> bool
     [for_all2] and [equal] differ: [equal eq xs ys] can
     be true only if [xs] and [ys] have the same length.
 
-    At least one of the sequences [xs] and [ys] must be finite.
+    May not terminate if at least one of the sequences [xs] and [ys] is
+    infinite.
```

Comment:
Is that true? Naively this seems incompatible with the remark above that iteration stops when either sequence is exhausted.

Reply:
You are right, my mistake. Fixed here and in all other places.